### ISSUE TITLE
[Accessibility] Python Native repl does not announce executed output

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#233237]
CC @meganrogge

When you execute code in Python native repl, the executed output is not announced to screen reader via aria alert.

## Reproducible Steps

1. Enable screen reader

2. Create Python native repl

3. In the input, type `print("hello")`

4. Hit Enter to execute

## Current Behavior

Executed result is not announced.

## Expected  Behavior

The executed result needs to be announced via aria alert.

**Comment by amunger** [comment]
@jooyoungseo - My plan was to just auto focus the item that was added, but from there, you would need to open the a11y view to get the output. Should this aria-alert be done instead of that? Or perhaps as an alternate?

related issue: https://github.com/microsoft/vscode/issues/229182

**Comment by jooyoungseo** [comment]
@amunger Moving focus after execution may cause some confusion. We don't move focus for Jupyter Notebooks and Copilot Chat, either. I would rather use aria-live alert to announce the executed output.

**Comment by amunger** [comment]
Thanks for the feedback, that makes sense.

**Comment by amunger** [comment]
verification steps: 
 
3. Get the python extension and have python installed locally
4. set `"editor.accessibilitySupport": "on"`
5. have a screen reader running
6. open a python file and use shift+enter to execute code with output, like `print('hello'), in the native REPL
7. ensure the screen reader announces "hello", or whatever the output was, through an aria alert

**Comment by aiday-mar** [comment]
Hi I am not sure if I am testing correctly. I do not hear the word 'hello' from voice over on Mac, even after clearing the cell output. You can see below the screen recording with the exact text from voice over. 

I have the pylance, python and python debugger extensions from Microsoft installed. 

https://github.com/user-attachments/assets/d9536fa7-7588-4829-b267-a05b9e08c605

**Comment by anthonykim1** [comment]
Thanks for testing. Does the behavior change if you do 
```
print('hello')
```
instead of defining hi variable later?

**Comment by dbaeumer** [comment]
@aiday-mar can you please answer.

**Comment by aiday-mar** [comment]
Hi yes just got down to answering. After changing print(hi) to print('hello') it still does not read hello when the cell output is generated. It says `Notebook Cell Completed`

https://github.com/user-attachments/assets/510f6bdb-7c5c-4ef0-9e6f-4275a52446cd

**Comment by amunger** [comment]
Some machines have issues with aria alerts, so please check another instance of something being read through an alert such as the copilot chat response.

If the copilot chat response is read in the screen reader, but the result of the REPL isn't, then re-open this issue.
